docs/openapi/schemas.yaml

```diff
@@ -29,6 +29,174 @@ URL:
   type: string
   format: url
   example: 'https://www.example.com'
+SlackMentionConfig:
+  type: object
+  additionalProperties: true
+  properties:
+    slack:
+      type: array
+      items:
+        type: string
+AlertConfig:
+  type: object
+  additionalProperties: true
+  properties:
+    type:
+      description: The type of the alert
+      type: string
+    byOrg:
+      description: Whether the alert should be sent by organization
+      type: boolean
+    mentions:
+      description: The Slack mentions to include in the alert
+      type: array
+      items:
+        $ref: '#/SlackMentionConfig'
+AlertsConfig:
+  type: array
+  items:
+    $ref: '#/AlertConfig'
+SlackConfig:
+  type: object
+  additionalProperties: true
+  properties:
+    channel:
```

Comment:
What format are these values in? For example, with the following channel URL: https://adobe.enterprise.slack.com/archives/C05B53SRX7V

Is channel `C05B53SRX7V` or `#franklin-spacecat`?

Is workspace `adobe.enterprise.slack.com`?

Reply:
@blefebvre the slack web api supports both the ID and name for specifying the channel. I think we should store the ID as it is more permanent than the name. The name can be looked up via the `conversations.list` API of slack if needed.

 The workspace is encoded in the token provisioned for the slack client and we'll probably reference the token by the workspace name. 

right @ekremney ?